Apps/Src/Voltage.c

```diff
@@ -18,6 +18,7 @@ static cell_asic *Minions;
 static OS_MUTEX Voltage_Mutex;
 static uint16_t VoltageVal[NUM_BATTERY_MODULES]; //Voltage values gathered
 static uint32_t openWires[TOTAL_VOLT_WIRES];
+static long open_wire_mask = 0;
```

Comment:
Why is this a `long` and not `uintXX_t`? 

Also, why `open_wire_mask` instead of `openWireMask`? (I realize our variable naming conventions are all over the place in this file, so I don't care too much about this)

Reply:
I made it a long because the data type it is being "&" with was a long, `Minions[i].system_open_wire`.